src/repo/github.js

```diff
@@ -1,6 +1,7 @@
 const url = require('url')
 const pify = require('pify')
 const request = pify(require('request'))
+const utilUrl = require('../util/url')
```

Comment:
```suggestion
const { isValidHttpUrl } = require('../util/url')
```
Or (to make use of your changes on https://github.com/all-contributors/all-contributors-cli/pull/268/files#diff-58b77d9dc33107f39855f1d42a64fa9e)
```suggestion
const { isValidHttpUrl } = require('../util').url;
```

Reply:
Done